src/entity_system.cpp

```diff
@@ -0,0 +1,1045 @@
+#include "entity_system.h"
```

Comment:
Did anything actually change in this file? Not sure why it says its a new file

Reply:
O: it was copied to the wrong directory! instead of src/map/src ------> src/
Needs to be deleted. sorry!

Reply:
Oh didn't notice the path